collection.go

```diff
@@ -637,7 +651,16 @@ func (c *Collection) SetVersion(db DB, version int64) error {
 
 func (c *Collection) createTable(db DB) error {
 	schema, _ := c.schemaTableName()
-	if schema != "public" {
+
+	if c.sqlAutocreateSchemaDisabled {
```

Comment:
Do we really need this option? Why can't we always check if schema exists and return if it does exist? Then we can remove this option.

Reply:
Indeed, that would be a good solution too.

The only downside I'm seeing is that you would get a `ERROR:  permission denied for database ${db_name}` instead of a presumably nicer `schema ${schema_name} does not exist; did you create it beforehand?` when the schema does not exist and you don't have the permissions to create it.

It's acceptable I guess to keep things simple.

I've pushed a new version following your guidelines, please let me know what you think of it.